certora/specs/methods/IAccessControlDefaultAdminRules.spec

```diff
@@ -0,0 +1,36 @@
+import "./IERC5313.spec"
+
+methods {
```

Comment:
I removed the functions and it's working, however, I found interesting that I kept coming back to the `methods` block for reference. Although I'm okay removing them, I think it somewhat improves my developer experience because I know in what order I'm trying to make a spec.